aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -533,6 +540,15 @@ def _resolve(self) -> ResponseBuilder:
         logger.debug(f"No match found for path {path} and method {method}")
         return self._not_found(method)
 
+    def _remove_prefix(self, path: str) -> str:
+        """Remove the configured prefix from the path"""
+        if self._strip_prefixes:
```

Comment:
that's gonna fail if someone passes a `""` or some other type

Reply:
correction, `[""]` works because of the way you're handling the slicing. It wouldn't if another type is sent, for example:

app = ApiGatewayResolver(strip_prefixes=[os.getenv("ENV_VAR_I_DIDNT_SET")]) # [None]

Reply:
ok i added many different invalid variations :)